tests/integ/modin/series/test_str_accessor.py

```diff
@@ -238,6 +238,44 @@ def test_str___getitem___string_key():
         snow_ser.str["a"]
 
 
+@pytest.mark.parametrize(
+    "pat",
+    [
+        "",
+        "xyz",
+        "uvw",
+        "%_.*?|&^$",
+        r"x.[za]",
+        r"(.?:abc|xyz)[^abcxyz]",
+        r"a|b|c",
+    ],
+)
+@pytest.mark.parametrize("case", [True, False])
+@pytest.mark.parametrize("flags", [0, re.IGNORECASE])
+@pytest.mark.parametrize("na", [None, np.nan, native_pd.NA, True, False])
+@sql_count_checker(query_count=1)
+def test_str_match(pat, case, flags, na):
+    native_ser = native_pd.Series(TEST_DATA)
+    snow_ser = pd.Series(native_ser)
+    eval_snowpark_pandas_result(
+        snow_ser,
+        native_ser,
+        lambda ser: ser.str.match(pat, case=case, flags=flags, na=na),
+    )
+
+
+@pytest.mark.parametrize("na", [1, "klm", datetime.date(2019, 12, 4), [True]])
+@sql_count_checker(query_count=0)
+def test_str_match_invlaid_na(na):
```

Comment:
```suggestion
def test_str_match_invalid_na(na):
```
spelling nit

Reply:
@sfc-gh-helmeleegy I think this got automerged, can you make this change in a follow-up PR (there are a few other string functions I think you're working on too)